CSA.md

```diff
@@ -1,91 +1,93 @@
+      (b) Upon <span class="coverpage_link">Customer's</span> request, <span class="coverpage_link">Provider</span> will delete <span class="coverpage_link">Customer Content</span> within 60 days of termination.
 
       (c) Each Recipient will return or destroy Discloser's Confidential Information in its possession or control.
 
       (d) <span class="coverpage_link">Provider</span> will submit a final invoice for all outstanding fees accrued before termination and <span class="coverpage_link">Customer</span> will pay the invoice according to Section 5 (Payment & Taxes).
 
     6.5 <span class="header_3">Survival.</span>  
 
-      (a) The following sections will survive expiration or termination of the Agreement: Section 1.4 (Feedback and Usage Data), Section 2.1 (Restrictions on <span class="coverpage_link">Customer</span>), Section 5 (Payment & Taxes) for fees accrued or payable before termination, Section 6.4 (Effect of Termination), Section 6.5 (Survival), Section 7 (Representations & Warranties), Section 8 (Disclaimer of Warranties), Section 9 (Limitation of Liability), Section 10 (Indemnification), Section 11 (Insurance) for the time period specified, Section 12 (Confidentiality), Section 13 (Reservation of Rights), Section 14 (General Terms), and Section 15 (Definitions).  
+      (a) The following sections will survive expiration or termination of the Agreement: Section 1.6 (Feedback and Usage Data), Section 2.1 (Restrictions on Customer), Section 5 (Payment & Taxes) for fees accrued or payable before expiration or termination, Section 6.4 (Effect of Termination), Section 6.5 (Survival), Section 7 (Representations & Warranties), Section 8 (Disclaimer of Warranties), Section 9 (Limitation of Liability), Section 10 (Indemnification), Section 11 (Insurance) for the time period specified, Section 12 (Confidentiality), Section 13 (Reservation of Rights), Section 14 (General Terms), Section 15 (Definitions), and the portions of a Cover Page referenced by these sections  
 
       (b) Each Recipient may retain Discloser's Confidential Information in accordance with its standard backup or record retention policies maintained in the ordinary course of business or as required by Applicable Laws, in which case Section 4 (Privacy & Security) and Section 12 (Confidentiality) will continue to apply to retained Confidential Information.
 
 7. ## Representations & Warranties  
 
     7.1 <span class="header_3">Mutual.</span>  Each party represents and warrants to the other that: (a) it has the legal power and authority to enter into this Agreement, (b) it is duly organized, validly existing, and in good standing under the Applicable Laws of the jurisdiction of its origin, (c) it will comply with all Applicable Laws in performing its obligations or exercising its rights in this Agreement, and (d) it will comply with the <span class="coverpage_link">Additional Warranties</span>.
```

Comment:
Changing commas to semi colons
```suggestion
    7.1 <span class="header_3">Mutual.</span>  Each party represents and warrants to the other that: (a) it has the legal power and authority to enter into this Agreement; (b) it is duly organized, validly existing, and in good standing under the Applicable Laws of the jurisdiction of its origin; (c) it will comply with all Applicable Laws in performing its obligations or exercising its rights in this Agreement; and (d) it will comply with the <span class="coverpage_link">Additional Warranties</span>.
```